cprover_bindings/src/goto_program/symtab_transformer/transformer.rs

```diff
@@ -592,6 +592,11 @@ pub trait Transformer: Sized {
         Stmt::decl(transformed_lhs, transformed_value, Location::none())
     }
 
+    fn transform_stmt_deinit(&mut self, place: &Expr) -> Stmt {
+        let transformed_place = self.transform_expr(place);
+        Stmt::deinit(transformed_place, Location::none())
```

Comment:
Is it supposed to be location none?

Reply:
Yes, because the location is modified added-later on. Every `transform_stmt_*` method returns a none location, and the location is propagated by the parent function `transform_stmt`